### ISSUE TITLE
"create form" giving unauthorised on open source portal

### ISSUE THREAD

**Issue by devkcodes** [formio/formio#1820]
**Description**
I used the master branch code in local deployment without docker ( using mongodb and nodejs).
 
In master branch, after logging in as ADMIN for the first time, i can't create any form as I'm getting a 401 error. This same thing when tried on the old version (which has client folder instead of postal) is working fine as expected.  

branch : master

**To Reproduce**
1. User master branch code on local deployment without docker
2. login to postal and create a form
3. 401 error which creating form.

**Screenshots**

I believe there should be an isAdmin key in the jwt which is not being sent from the frontend (can be different issue  but this is what i got from few console logs in tokenHandler middleware)

**Comment by Devanshkcodes** [comment]
>    I am also encountering errors when trying to create a form however I am getting a 400 error. The error that I am getting is due to the payload not having name, title, and path even though I have provided those values in the textfields.


You are correct. After further investigating, I realized I had changed the database name in default.json while testing and did not clear the local storage thus sending request with wrong token. Regardless, the issue of  401 bad request persists while creating forms. As you mentioned, data filled in the form is not sent in the payload, thus the bad request error. Here are the screenshots
 
![Screenshot 2024-10-23 233829](https://github.com/user-attachments/assets/411b3da7-de70-422f-be0b-e8d683176a40)
![Screenshot 2024-10-23 233841](https://github.com/user-attachments/assets/69c436d0-2c00-429c-8a18-d94c57c2cc66)
![Screenshot 2024-10-23 233858](https://github.com/user-attachments/assets/214912f2-1eb7-4197-b110-664009d6c66d)

**Comment by devkcodes** [comment]
You are correct. After further investigating, I realized I had changed the database name in default.json while testing and did not clear the local storage thus sending request with wrong token. Regardless, the issue of 401 bad request persists while creating forms. As you mentioned, data filled in the form is not sent in the payload, thus the bad request error. Here are the screenshots

![Screenshot 2024-10-23 233829](https://github.com/user-attachments/assets/e87f7e01-eb30-48f3-b95f-647f80827bf8)
![Screenshot 2024-10-23 233841](https://github.com/user-attachments/assets/e08ca7ca-54e0-4b06-a2cf-e16bd622700c)
![Screenshot 2024-10-23 233858](https://github.com/user-attachments/assets/2f55b2dc-27d8-4dd8-ac13-5443dd0eb881)

**Comment by piotr-huk** [comment]
Please take a look at the issue in formio/react - https://github.com/formio/react/issues/580
I believe it's the same problem, which can be resolved by updating packages versions in the portal
